bedrock-devnet/devnet/__init__.py

```diff
@@ -33,88 +34,112 @@ def main():
     contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'contracts-bedrock')
     deployment_dir = pjoin(contracts_bedrock_dir, 'deployments', 'devnetL1')
     op_node_dir = pjoin(args.monorepo_dir, 'op-node')
-    ops_bedrock_dir=pjoin(monorepo_dir, 'ops-bedrock')
+    ops_bedrock_dir = pjoin(monorepo_dir, 'ops-bedrock')
+    deploy_config_dir = pjoin(contracts_bedrock_dir, 'deploy-config'),
+    devnet_config_path = pjoin(contracts_bedrock_dir, 'deploy-config', 'devnetL1.json')
 
     paths = Bunch(
       mono_repo_dir=monorepo_dir,
       devnet_dir=devnet_dir,
       contracts_bedrock_dir=contracts_bedrock_dir,
       deployment_dir=deployment_dir,
-      deploy_config_dir=pjoin(contracts_bedrock_dir, 'deploy-config'),
+      l1_deployments_path=pjoin(deployment_dir, '.deploy'),
+      deploy_config_dir=deploy_config_dir,
+      devnet_config_path=devnet_config_path,
       op_node_dir=op_node_dir,
       ops_bedrock_dir=ops_bedrock_dir,
       genesis_l1_path=pjoin(devnet_dir, 'genesis-l1.json'),
       genesis_l2_path=pjoin(devnet_dir, 'genesis-l2.json'),
+      allocs_path=pjoin(devnet_dir, 'allocs-l1.json'),
       addresses_json_path=pjoin(devnet_dir, 'addresses.json'),
       sdk_addresses_json_path=pjoin(devnet_dir, 'sdk-addresses.json'),
       rollup_config_path=pjoin(devnet_dir, 'rollup.json')
     )
 
     os.makedirs(devnet_dir, exist_ok=True)
 
+    if args.allocs:
```

Comment:
So `--allocs` means building the genesis?

Reply:
It technically isn't building the genesis file, its building the allocs field in the genesis file